packages/rest-api-client/src/client/types/index.ts

```diff
@@ -1,6 +1,8 @@
 export type AppID = string | number;
 export type RecordID = string | number;
+export type SpaceID = string | number;
 export type Revision = string | number;
```

Comment:
NITS: The `Revision` is the domain of App and Record, so it's better to belong to them.

```suggestion
export type AppID = string | number;
export type RecordID = string | number;
export type Revision = string | number;
export type SpaceID = string | number;
```